src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -20,22 +20,22 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                new Address("Blk 30 Geylang Street 29, #06-40"),
+                new ModuleCode("Blk 30 Geylang Street 29, #06-40"),
```

Comment:
Testing of new ModuleCode field may be more accurate if the input was changed to match the module code format.